themes/Default/Dark.qss

```diff
@@ -198,11 +198,16 @@ QMenu::item {
   border: 1px solid transparent;
   color: #d8d8d8;
   padding: 5px;
-  padding-right: 16px;
+  padding-right: 20px;
```

Comment:
Do we really want changes like this to our CSS or was this just part of a workaround to manually fix the layout issues that have been encountered in the original PR?

Reply:
No, I am currently investigating too little right-padding which seems to have been present in and before #5771. It seems like this would need another fix (See the ... of Local Volume Adjustment is literally on the line of the menu)

Reply:
See how in #5462, screenshot 2, "Change Comment..." has too little right padding?

Reply:
Qt stylesheets are a massive pain not gonna lie